tests/NBomber.IntegrationTests/FeedTests.fs

```diff
@@ -0,0 +1,41 @@
+module NBomber.IntegrationTests.FeedTests
+
```

Comment:
Can we have integration tests?
It's really important to just see that this feature is not breaking anything
(It's optional but you can use logger and write assets based on this. I did it for the cluster for example)

Reply:
I can check whether this feature isn't broken :), the other way round is more difficult. But I will see what I can do next day(s)

Reply:
I think you can just create a test with some basic feed, then run and then check stats

Reply:
I think you can just create a test with some basic feed, then run and then check stats

Reply:
added a test. OkCount is always less than values counted. No warm up. Can you explain?

Reply:
how much less?
I think I need to debug then.

Reply:
as an example: `32536 = 32481` --> count > stats.OkCount